modules/core/src/transactionBuilder.ts

```diff
@@ -920,10 +919,13 @@ exports.signTransaction = function(params) {
     }
 
     const chainPath = currentUnspent.chainPath;
-    if (rootExtKeyPath) {
-      const subPath = keychain.walletSubPath || '/0/0';
-      const path = keychain.path + subPath + chainPath;
-      privKey = rootExtKeyPath.deriveKey(path);
+    if (rootExtKey) {
+      let { walletSubPath = '/0/0' } = keychain;
+      if (walletSubPath === 'm') {
```

Comment:
Just to be clear, `m` will only be here as the only character and not with anything following it, right?

Reply:
yes, although on second though we should probably be a bit more flexible

Reply:
let's do https://github.com/BitGo/BitGoJS/pull/1347 first